community/samples/serving/helloworld-python-bentoml/README.md

```diff
@@ -0,0 +1,243 @@
+    ```
+
+2. This code defines how to train a classifier model with iris dataset and how to save
+  the model with BentoML. Run the following code:
+
+    ```python
+    from sklearn import svm
+    from sklearn import datasets
+
+    # import the class from the file that was created from the previous step
+    from iris_classifier import IrisCLassifier
+
+    if __name__ == "__main__":
+        # Load training data
+        iris = datasets.load_iris()
+        X, y = iris.data, iris.target
+
+        # Model Training
+        clf = svm.SVC(gamma='scale')
+        clf.fit(X, y)
+
+        # Create a iris classifier service instance
+        iris_classifier_service = IrisClassifier()
+
+        # Pack the newly trained model artifact
+        iris_classifier_service.pack('model', clf)
+
+        # Save the prediction service to disk for model serving
+        saved_path = iris_classifier_service.save()
+    ```
+
+3. Use BentoML CLI to check saved model's information.
+
+    ```shell
+    bentoml get IrisClassifier:latest
+    ```
+
+    Example:
+
+    ```shell
+    > bentoml get IrisClassifier:latest
+    {
+      "name": "IrisClassifier",
+      "version": "20200305171229_0A1411",
+      "uri": {
+        "type": "LOCAL",
+        "uri": "/Users/bozhaoyu/bentoml/repository/IrisClassifier/20200305171229_0A1411"
+      },
+      "bentoServiceMetadata": {
+        "name": "IrisClassifier",
+        "version": "20200305171229_0A1411",
+        "createdAt": "2020-03-06T01:12:49.431011Z",
+        "env": {
+          "condaEnv": "name: bentoml-IrisClassifier\nchannels:\n- defaults\ndependencies:\n- python=3.7.3\n- pip\n",
+          "pipDependencies": "bentoml==0.6.2\nscikit-learn",
+          "pythonVersion": "3.7.3"
+        },
+        "artifacts": [
+          {
+            "name": "model",
+            "artifactType": "SklearnModelArtifact"
+          }
+        ],
+        "apis": [
+          {
+            "name": "predict",
+            "handlerType": "DataframeHandler",
+            "docs": "BentoService API",
+            "handlerConfig": {
+              "orient": "records",
+              "typ": "frame",
+              "input_dtypes": null,
+              "output_orient": "records"
+            }
+          }
+        ]
+      }
+    }
+    ```
+
+4. Test run API server. BentoML can start an API server from the saved model. Use
+  BentoML CLI command to start an API server locally and test it with the `curl` command.
+
+    ```shell
+    bentoml serve IrisClassifier:latest
+    ```
+
+    In another terminal window, make `curl` request with sample data to the API server
+    and get prediction result:
+
+    ```shell
+    curl -v -i \
+    --header "Content-Type: application/json" \
+    --request POST \
+    --data '[[5.1, 3.5, 1.4, 0.2]]' \
+    127.0.0.1:5000/predict
+    ```
+
+## Building and deploying the sample
+
+BentoML supports containerlization a dockerfile for API server of the saved model.
```

Comment:
```suggestion
BentoML supports containerization using a Dockerfile for API server of the saved model.
```

I don't see where the Dockerfile is created here?

Reply:
Dockerfile is automatically generated as part of the saving model process.  I updated the readme to make it clear.  Thank you for the suggestion